docs/source/author/content_style_guide.rst

```diff
@@ -0,0 +1,231 @@
+Content Style Guide
+##############################################################################
+
+.. note::
+
+  This page is currently intended for internal staff only.
+
+
+.. contents:: Contents
+  :local:
+
+ 
+Writing Style
+==============================================================================
+
+When using pronouns in reference to a hypothetical person, gender neutral pronouns (they/their/them) should be used. 
+ 
+All documents (other than those for internal use only) must be written clearly and simply so that a non-expert is able to understand them.
+Preferably documents should be readable for students. 
+ 
+Any jargon used needs to be clearly explained. 
+
+
+Capitalisation Rules
+------------------------------------------------------------------------------
+
+In the majority of cases capitalisation should not be used for keywords and titles, with the following exceptions, where the phrase refers to a commonly used term that is often capitilised in the literature:
+
+- Computer Science
+- Computational Thinking
+- Digital Technologies (note that this is the correct form to refer to the subject area in NZ, with caps and plural; if it's referring to something other than the subject then use lower case e.g. "smartphones and other digital technologies", or even better, avoid the phrase e.g. "smartphones and other digital devices")
+- Sorting Network
+- Numeracy
+- Literacy
+ 
+The following wouldn't be capitalised:
+binary number(s), digits, binary digits
+ 
+
+
+Google Drive Folder Structure
+==============================================================================
+
+Including Images
+------------------------------------------------------------------------------
+
+All images included in a document have a comment which links to the original image file in a specified Google Drive Folder.
```

Comment:
This part "in a specified Google Drive Folder" isn't really needed if it's a direct link. But if this statement is left there, then a link to what the "specified" part should be made.